enhancements/update/configurable-clusterversion-signature-stores.md

```diff
@@ -0,0 +1,194 @@
+---
+title: configurable-clusterversion-signature-stores
+authors:
+  - "@PratikMahajan"
+reviewers:
+  - "@LalatenduMohanty"
+  - "@wking"
+approvers:
+  - "@LalatenduMohanty"
+api-approvers:
+  - "@deads2k"
+creation-date: 2023-09-27
+last-updated: 2023-10-31
+tracking-link:
+  - https://issues.redhat.com/browse/OTA-916
+---
+
+# ClusterVersion has option to replace default signature stores
```

Comment:
nit: The current proposal is allowing for extending the set of signature stores to include additional sources.  It does not (currently) support removing the default sources from the set of stores.  So we might want to swap out "replace" and instead use "extend" or something here.

Reply:
> does not (currently) support removing the default sources from the set of stores

Is this something that you see being supported in a future version of this feature? If so, I would like to make sure the API is set up to account for that eventual improvement